front/context/src/AccountsContext.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { web3Accounts, web3Enable, isWeb3Injected } from '@polkadot/extension-dapp';
+import React, { useState, createContext, useEffect } from 'react';
+import { InjectedAccountExt } from './types';
+
+export const AccountsContext = createContext({
+  injectedAccounts: [] as InjectedAccountExt[]
+})
+
+interface Props {
+  children: any;
+}
+
+export function AccountsContextProvider (props: Props): React.ReactElement {
+  const { children } = props;
+  const [injectedAccounts, setInjected] = useState<InjectedAccountExt[]>([] as InjectedAccountExt[]);
```

Comment:
Is the `as ...` necessary here?

Reply:
maybe not the best way around it but the tsc error here was: `cannot assign never[] to InjectedAccountExt[]`